test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/MalformedPageDirective_DesignTime.codegen.cs

```diff
@@ -13,10 +13,6 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_MalformedP
     {
         #pragma warning disable 219
         private void __RazorDirectiveTokenHelpers__() {
-        ((System.Action)(() => {
-global::System.Object __typeHelper = "foo;
```

Comment:
The directive token is no longer valid (we're validating now) so it's not output into the `DirectiveBlock`. Instead the token is treated as markup in the rest of the page. With this approach directive authors never see invalid tokens.